src/main/java/uk/gov/pay/products/exception/mapper/PaymentCreationExceptionMapper.java

```diff
@@ -8,6 +8,9 @@
 import javax.ws.rs.core.Response;
 import javax.ws.rs.ext.ExceptionMapper;
 
+import static org.eclipse.jetty.http.HttpStatus.FORBIDDEN_403;
+import static org.eclipse.jetty.http.HttpStatus.INTERNAL_SERVER_ERROR_500;
```

Comment:
Think I’d prefer to get the constants from `javax.ws.rs.core.Response.Status` or similar rather than a third-party library (yes, I know Java EE is now technically a third-party library!).

Reply:
updated